addons/website_membership/controllers/main.py

```diff
@@ -103,7 +103,7 @@ def members(self, membership_id=None, country_name=None, country_id=0, page=1, *
 
         # get google maps localization of partners
         google_map_partner_ids = []
-        if request.env.ref('website_membership.opt_index_google_map').customize_show:
+        if request.website.viewref('website_membership.opt_index_google_map').active:
```

Comment:
`.customize_show` to `.active` to backport, couldnt test the module anyway even on runbot, throws an error on unexisting column `website_published`

Reply:
Done at https://github.com/odoo/odoo/pull/29414